index.html

```diff
@@ -239,8 +144,11 @@ <h2>Contact me</h2>
                 
             </div>
         </div>
-        
     </footer>
+
+    <!-- Modal -->
+    <div id="modal-section"></div>
+    
```

Comment:
[ ] I can't see the close button. kindly add it when the user clicks it the popup disappears.
Figma design : 
![popupdesign](https://user-images.githubusercontent.com/62156607/208743098-9c401117-9d6e-4336-b6ec-1981e0f77ca2.PNG)
Your popup : 
![popupte3ou](https://user-images.githubusercontent.com/62156607/208743167-be9529d4-5c8f-46c0-8a0d-82d4f035b52f.PNG)

Reply:
Thank You @nedjwab. So I can actually see the missing content you mention here on my end, I really don't know why you can not see them.

could it be because I updated the changes after requesting for review?

![image](https://user-images.githubusercontent.com/43586022/208743999-fab8028f-a2da-4a8b-814d-c5aeba0de8a2.png)